app/src/main/java/org/wikipedia/activitytab/ActivityTabViewModel.kt

```diff
@@ -235,6 +235,20 @@ class ActivityTabViewModel() : ViewModel() {
         return listOf(historyEntryPagingSource, readingListPagingSource, userContribPagingSource)
     }
 
+    fun getTotalEditsCount(): Int {
+        return when (val currentState = _impactUiState.value) {
+            is UiState.Success -> currentState.data.totalEditsCount
+            else -> 0
+        }
+    }
+
+    fun isDonationUnknown(): Boolean {
```

Comment:
The function name is about whether the donation is `unknown` or not, but the actual logic is being done in the opposite way.